common/src/main/java/com/datastrato/gravitino/dto/rel/ColumnDTO.java

```diff
@@ -37,6 +38,12 @@ public class ColumnDTO implements Column {
   @JsonProperty("autoIncrement")
   private boolean autoIncrement = false;
 
+  @JsonProperty("defaultValue")
+  @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
What's the meaning of this annotation?

Reply:
comment added.

"the NON_EMPTY annotation is used to avoid serializing the defaultValue field if it is not set or null."